news/2021-03-18-community-contributors-2020.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: post
+title: "Community Contributors: 2020"
+date: 2021-03-18 10:30:00 +0000
+---
+
+Today, we celebrate some of the people who make the osu! community truly tick by awarding them our highest honour: the badge of a Community Contributor.
+
+![](/wiki/shared/news/2021-03-18-community-contributors-2020/title.jpg)
+
+osu! is one of the rare bastions still left in the modern-day internet where community is everything. From day one, we've been built up by the tireless efforts of hundreds of individuals who selflessly dedicate themselves towards making things more awesome for everyone.
+
+It is by their efforts that the osu! of today exists, and we thank them by periodically seeking out the exemplars among us and granting them the Community Contributor badge, a signet of their enduring dedication and commitment to the osu! community.
+
+![Community Contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg)
+
+Today, we welcome more into the fold, and perhaps some that should have been welcomed long ago.
+
+We present to you, the latest of the greatest:
+
+## Contributors
+
+[![](/wiki/shared/news/2021-03-18-community-contributors-2020/hallowatcher.jpg)](https://osu.ppy.sh/users/1874761)
+
+**hallowatcher** has had a long and storied history with osu!, but is most renowned for his creation of the now-ubiquitous 'pickems' games that accompany the official World Cups and a number of community tournaments as well. Pickems have since become something of a sensation, attracting well over 5000+ unique users and over 40 variants hosted, and several have even been the recipients of official World Cup profile badges, all while remaining a top tier tournament competitor himself.
```

Comment:
> several have even been the recipients of official World Cup profile badges

if this refers to the "5000+ unique users", switch it with "40 variants hosted". currently sounds like the pickems receive wc badges

> all while remaining a top tier tournament competitor himself.

i assume this is about hallowatcher, but the sentence is about pickems so it reads super weird

Reply:
jostled the wording around a bit